docs/process/roadmap.md

```diff
@@ -3,45 +3,48 @@ The following timeline is derived from [ICU4X 1.0 PRD](./prd.md) and is optimize
 
 * [x] 2020
   * [x] October
-	  * [x] 0.1 Release
+	  * [x] 0.1 Release (#204)
   * [x] November/December
-	  * [x] Post 0.1 clean up and maintenance cycle
+	  * [x] Post 0.1 clean up and maintenance cycle ([Milestone](https://github.com/unicode-org/icu4x/milestone/7))
 * [ ] 2021
 	* [x] January/February
 		* [x] Locale
-			* [x] Likely Subtags
+			* [x] Likely Subtags (#417)
 		* [x] DateTime
-			* [x] Day Periods
-			* [x] Timezones
-			* [x] Components Bag
+			* [x] Day Periods (#435)
+			* [x] Timezones (#418)
+			* [x] Components Bag (#481)
 		* [x] Unicode Set
-			* [x] L3a
+			* [x] L3a (#478)
 	* [ ] March/April
-		* [ ] FFI/WASM exploration
-		* [ ] Static Data Provider
-		* [ ] Unicode Set L3b
-		* [ ] 0.2 Release
+		* [ ] FFI/WASM exploration (#207)
+		* [ ] Static Data Provider (#632)
```

Comment:
```suggestion
		* [ ] FFI/WASM exploration (#398)
		* [ ] Static Data Provider (#78)
```

Also, "Static Data Provider" should be in 0.3, not 0.2

Reply:
I didn't mean the order to impose the release, the fact that work is being done on data provider in April and that we release 0.2 in April may be coincidental.
But to make it a bit more clear, I moved data provider past 0.2 in order.